HackYourFuture-CPH/rate-my-cv
# 34 uploading file

# Description

This story is about a function that when invoked with a file as a argument will upload the file. It returns a url for accessing that file! 

Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/34

# How to test?

For making a test you should first creat a project in firebase and a database in there as well. You should change the firebase Config here in the code based on the configure parameters in firebase. Afterward give a file to the function and in the result you can give a Url of that updated file. 
 
# Checklist
